Core/clim/text-editor-gadget.lisp

```diff
@@ -230,7 +230,7 @@ cause the activate callback to be called."))
                                :disarmed-callback disarmed-callback
                                :activation-gestures activation-gestures
                                :activate-callback activate-callback
-                               :scroll-bars scroll-bars
+                               ;:scroll-bars scroll-bars
```

Comment:
good catch! please remove the initarg (instead of commenting it).

Reply:
done

Reply:
thanks, I'm going to squash this (since there are only two commits, for a more complex pull request I'd ask you to do that).